frontend/openchat-shared/src/constants.ts

```diff
@@ -1 +1,10 @@
 export const ONLINE_THRESHOLD = 120;
+export const MAX_MESSAGES = 100;
+export const MAX_EVENTS = 500;
+export const MAX_MISSING = 30;
+export const OPENCHAT_BOT_USER_ID = "zzyk3-openc-hatbo-tq7my-cai";
+export const OPENCHAT_BOT_USERNAME = "OpenChatBot";
+export const OPENCHAT_BOT_AVATAR_URL = "assets/robot.svg";
+
+// downlink is the effective bandwidth estimate in megabits per second, rounded to the nearest multiple of 25 kilobits per seconds.
+export const MIN_DOWNLINK = 2;
```

Comment:
I reckon this number should be lower... say 0.5

Reply:
It's a bit unclear what it actually _means_ - it doesn't actually seem to mean what I thought it did. I will try to pin it down exactly